community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v3_4/SemanticIndexAcceptanceTest.scala

```diff
@@ -25,10 +25,23 @@ import org.neo4j.cypher.ExecutionEngineFunSuite
 import org.neo4j.values.storable._
 import org.scalacheck.Gen
 import org.scalacheck.Arbitrary.arbitrary
+import org.scalatest.matchers.{MatchResult, Matcher}
 import org.scalatest.prop.PropertyChecks
 
 import scala.collection.JavaConversions._
 
+/**
+  * After a failure, do this to reproduce with the actual values that caused the error:
+  *
+  * place a single call to testOperator after the for-loop. As an example
+  * {{{
+  * testOperator("=", ValueSetup[DateTimeValue]("dateTimes", Gen.oneOf(Seq(
+  *     DateTimeValue.parse("1901-12-13T20:45:52Z", null),
+  *     DateTimeValue.parse("1901-12-13T20:45:52Z[Europe/Brussels]", null)
```

Comment:
is specifying both `Z` and a named zone allowed by the parser? We should disallow that.

Reply:
`UTC = ZoneOffset.ofTotalSeconds(0)``from ZoneOffset.java